cdn/dev/keyboard-search/dedicated-landing-pages.js

```diff
@@ -0,0 +1,66 @@
+//
+// Meta search results for dedicated landing pages
+//
+
+const dedicatedLandingPages = [
+  {
+    id: "/amharic",
+    terms: ['amharic', 'ge\'ez', 'geez', 'ethiopic'],
+    description: "Amharic keyboards recommended by the Keyman team",
+    name: "Amharic Keyboards Home Page"
+  },
+  {
+    id: "/burmese",
+    terms: ['burmese', 'myanmar'],
+    description: "Burmese (Myanmar) keyboards recommended by the Keyman team",
+    name: "Burmese Keyboards Home Page"
+  },
+  {
+    id: "/cameroon",
+    terms: ['cameroon'],
+    description: "Cameroon keyboards recommended by the Keyman team",
+    name: "Cameroon Keyboards Home Page"
+  },
+  {
+    id: "/greek",
+    terms: ['greek', 'ancient greek', 'biblical greek', 'attic greek', 'classical greek', 'polytonic', 'polytonic greek'],
+    description: "Biblical and Classical Greek keyboards recommended by the Keyman team",
+    name: "Classical Greek Keyboards Home Page"
+  },
+  {
+    id: "/ipa",
+    terms: ['ipa', 'phonetic', 'phonetic alphabet', 'international phonetic alphabet', 'transcription'],
+    description: "IPA (International Phonetic Alphabet) keyboards recommended by the Keyman team",
+    name: "IPA Keyboards Home Page"
+  },
+  {
+    id: "/sinhala",
```

Comment:
I see two landing pages (garp looks to be the default)
/sinhala/garp
/sinhala/sipon

Reply:
/sinhala redirects to /sinhala/garp. I am comfortable with this -- in the future we may rearrange. I'm not trying to reorganise the site much at this point, just translate the existing layout to the new one.